src/pages/Graph/SummaryPanelTraceDetails.tsx

```diff
@@ -170,6 +162,14 @@ class SummaryPanelTraceDetails extends React.Component<Props, State> {
                 this.renderSpan(nodeName + '.' + node.namespace, spans[this.state.selectedSpan])}
             </p>
           )}
+          {tracesDetailsURL && jaegerTraceURL && (
+            <>
+              <br />
+              <a href={jaegerTraceURL} target="_blank" rel="noopener noreferrer">
+                View Trace in Tracing <ExternalLinkAltIcon size="sm" />
```

Comment:
I'm OK with the bottom-positioning but again, I'd use 'Show'.  Also, because we know we're looking at a trace I think we can be less wordy, so perhaps just `Show in Tracing`.